### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by domitilacrispim**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

Fixes #{ISSUE_NUMBER}

**By submitting this pull request I confirm I've read and complied with the
below declarations.**

- [ ] I have read the [Contribution guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [ ] I have followed the [commit guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#write-good-commit-messages) for this project.
- [ ] My code follows the [standard code structure](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#samples-code).
- [ ] This pull request has a descriptive title. For example, `{Tag}: Add {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [ ] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.
- [ ] This pull request shall only be reviewed and merged once all the checks passes. No maintainer or supporter shall be obliged to review it before this condition is met.
- [ ] I have mentioned the issue number correctly (with hyperlink) in this pull request description.

After you submit your pull request, **DO NOT** click the 'Update Branch' button.

### CODE REVIEW

File: selection.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#include<bits/stdc++.h>
+int main (){
+	int stack[1000], op, top=9999, po, aux;
+	printf("Enter the length of vector\nThe system suports until 1000 elements!\n");
+	scanf("%d", &op);
+	printf("Enter the elements!\n");
+
```
Comment (sangamcse) [inline selection.cpp]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgo77r2ps/selection.cpp
+++ b/tmp/tmpgo77r2ps/selection.cpp
@@ -4,7 +4,7 @@
 	printf("Enter the length of vector\nThe system suports until 1000 elements!\n");
 	scanf("%d", &op);
 	printf("Enter the elements!\n");
-	
+
 	for ( int i=0; i<op; i++){
 		scanf("%d", &stack[i]);
 	}
```

File: selection.cpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+int main (){
+	int stack[1000], op, top=9999, po, aux;
+	printf("Enter the length of vector\nThe system suports until 1000 elements!\n");
+	scanf("%d", &op);
+	printf("Enter the elements!\n");
+	
+	for ( int i=0; i<op; i++){
+		scanf("%d", &stack[i]);
+	}
+
```
Comment (sangamcse) [inline selection.cpp]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgo77r2ps/selection.cpp
+++ b/tmp/tmpgo77r2ps/selection.cpp
@@ -8,7 +8,7 @@
 	for ( int i=0; i<op; i++){
 		scanf("%d", &stack[i]);
 	}
-	
+
 	for ( int i=0; i<op; i++){
 		top=stack[i];
 		po=i;
```

File: selection.cpp [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+		scanf("%d", &stack[i]);
+	}
+	
+	for ( int i=0; i<op; i++){
+		top=stack[i];
+		po=i;
+		for ( int j=i; j<op; j++){
+			if(stack[j]<top){
+				top=stack[j];
+				po=j;
```
Comment (sangamcse) [inline selection.cpp]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgo77r2ps/selection.cpp
+++ b/tmp/tmpgo77r2ps/selection.cpp
@@ -15,7 +15,7 @@
 		for ( int j=i; j<op; j++){
 			if(stack[j]<top){
 				top=stack[j];
-				po=j;		
+				po=j;
 			}
 		}
 		aux=stack[i];
```

File: selection.cpp [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+		for ( int j=i; j<op; j++){
+			if(stack[j]<top){
+				top=stack[j];
+				po=j;		
+			}
+		}
+		aux=stack[i];
+		stack[i]=stack[po];
+		stack[po]=aux;
+
```
Comment (sangamcse) [inline selection.cpp]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgo77r2ps/selection.cpp
+++ b/tmp/tmpgo77r2ps/selection.cpp
@@ -21,7 +21,7 @@
 		aux=stack[i];
 		stack[i]=stack[po];
 		stack[po]=aux;
-		
+
 	}
 	for( int i=0; i<op; i++)
 		printf(" %d ", stack[i]);
```

Comment (domitilacrispim) [reply selection.cpp]: Sorry, i dont speak english and i dont understand can you talk in spanish?

**Pull request closed without merging**